logger/redacting_formatter.go

```diff
@@ -0,0 +1,48 @@
+package logger
+
+import (
+	"regexp"
+
+	"github.com/sirupsen/logrus"
+)
+
+type RedactionOption struct {
+	Field       string
+	Regexp      *regexp.Regexp
+	ReplaceWith string
+}
+
+type RedactingFormatter struct {
+	logrus.Formatter
+	fields []*RedactionOption
+}
+
+func (f *RedactingFormatter) Format(entry *logrus.Entry) ([]byte, error) {
```

Comment:
Even if covered in `logger_test.go`, I would add a dedicated test for this method, and cover more cases (as per the "[REDACTED]" replacement).
What do you think ?

Reply:
I did think about this. I hesitated, but you're right it's probably better so I'll add one.
Thanks for your input on this one!

Reply:
perfect :heart: